cmd/app/command/device-list.go

```diff
@@ -18,11 +18,20 @@ package command
 
 import (
 	goflag "flag"
+	"fmt"
+	"html/template"
+	"os"
 
 	"github.com/openebs/node-disk-manager/cmd/controller"
 	"github.com/spf13/cobra"
 )
 
+// defaultDeviceList template use to print list of device
+const defaultDeviceList = `{{if .Items}}{{printf "%-45s" "NAME"}}{{printf "%-20s" "MODEL"}}{{printf "%-20s" "VENDOR"}}{{printf "%-25s" "SERIAL"}}{{printf "%-10s" "STATUS"}}{{printf "%-10s" "PATH"}}CAPACITY
```

Comment:
Can this be split across lines for better readability.

Reply:
@shovanmaity Check if `{{- some templating code }}` can be used to address @kmova 's comment.
Hyphen `{{-` will remove the whitespace from the left.

Reply:
thanks @AmitKumarDas . I have updated template that way.